core/shared/src/main/scala/io/udash/properties/PropertySeqCombinedReadableSeqProperty.scala

```diff
@@ -0,0 +1,37 @@
+package io.udash.properties
+
+import com.avsystem.commons.ISeq
+import io.udash.properties.Properties.{Patch, Property, ReadableProperty}
+import io.udash.properties.seq.AbstractReadableSeqProperty
+import io.udash.utils.Registration
+
+private[properties] final class PropertySeqCombinedReadableSeqProperty[A](value: ISeq[Property[A]])
+  extends AbstractReadableSeqProperty[A, ReadableProperty[A]] {
+
+  override val id: PropertyId = PropertyCreator.newID()
+  override protected[properties] val parent: ReadableProperty[_] = null
+
+  private val children = CrossCollections.createArray[ReadableProperty[A]]
+
+  value.foreach(property => {
+    children.+=(property.transform(value => value))
```

Comment:
This type of transform is not a good way to ensure that no one can access mutable API with pattern matching. I know that there is no better way now, but it's time to provide it. I think method `_.readonly` returning a minimal wrapper around the property will be much better. (Ping @ddworak)

Reply:
True, one can always pattern match on the property at runtime and find out it has some nice mutable features.

Reply:
#196